### PR TITLE
Analyze with fatal infos on both `stable` and `beta`

### PR DESCRIPTION
**Pull request by domesticmouse**
Keep the flutter/samples repo analyzer clean on both `stable` and `beta` channels to make quarterly releases cleaner.

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/samples/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: add_to_app/android_view/android_view/gradlew.bat [hunk 1]
```diff
@@ -1,84 +1,84 @@
...
+:fail
+rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
+rem the _cmd.exe /c_ return code!
+if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
+exit /b 1
+
+:mainEnd
+if "%OS%"=="Windows_NT" endlocal
+
+:omega
```
Comment (craiglabenz) [inline add_to_app/android_view/android_view/gradlew.bat:84]: This diff looks frivolous. Is there a possible newlines conflict?

File: tool/flutter_ci_script_shared.sh [hunk 2]
```diff
@@ -12,11 +12,7 @@ function ci_projects () {
         flutter pub get
 
         # Run the analyzer to find any static analysis issues.
-        if [ "$channel" == 'stable' ]; then
-            dart analyze --fatal-infos
-        else
-            dart analyze
-        fi
+        dart analyze --fatal-infos
```
Comment (craiglabenz) [inline tool/flutter_ci_script_shared.sh:15]: Something tells me this is the primary part of interest

**Review by craiglabenz — APPROVED** [review]
Seems good, but we should review what's going on with all the .bat files which are marked as entirely changed despite having a lot of (all?) shared lines

File: tool/flutter_ci_script_shared.sh (hunk 2 above)
Comment (domesticmouse) [reply tool/flutter_ci_script_shared.sh:15]: Yup, thats the real change

File: add_to_app/android_view/android_view/gradlew.bat (hunk 1 above)
Comment (domesticmouse) [reply add_to_app/android_view/android_view/gradlew.bat:84]: I modified [.gitattributes](https://github.com/flutter/samples/blob/master/.gitattributes) which now enforces end of line configuration for a variety of files. When these files get touched, the end of line setting is then enforced by git.

**Comment by domesticmouse** [comment]
The fun bbit is [the beta branch failure](https://github.com/flutter/samples/runs/4572651002?check_suite_focus=true#step:5:438). This is tracked in https://github.com/flutter/samples/issues/963

**Comment by domesticmouse** [comment]
The only out standing red in CI is cleaned up in https://github.com/flutter/samples/pull/978

**Pull request merged**